packages/cli/test/fixtures/connected-diagram.mmd

```diff
@@ -0,0 +1,6 @@
+---
+title: My cool flowchart
+id: my-test-document-id
```

Comment:
I'm slightly leaning towards having a URL here instead of an ID, e.g. something like:

```yaml
id: https://mermaidchart.com/app/diagrams/my-test-document-id
```

The benefits are:
  - It's more RESTful
  - It's much easier for users to just click the link and edit it on MermaidChart.com (assuming they have the correct permissions)
  - It makes it easier to see if a diagram has been synced to MermaidChart.com, or an on-premises Mermaid Chart instance.

The downsides are:
  - Parsing will be a bit more complicated
  - It may look a bit messy, since then we're adding more text to the diagram.

Reply:
Will be fixed by #20!